app/models/katello/presenters/sync_status.rb

```diff
@@ -0,0 +1,69 @@
+module Katello
+  module Presenters
+    class SyncStatus
+      include ::ActionView::Helpers::DateHelper
+      include ::Katello::TranslationHelper
+      STATUS_VALUES = {   :stopped      => _("Syncing Complete."),
```

Comment:
Could be put a blank lin ebetween the includes and the constant definition? Any reason for the initial blank spaces before :stopped ?